keps/sig-cluster-lifecycle/kubeadm/20190122-Certificates-copy-for-kubeadm-join--control-plane.md

```diff
@@ -90,6 +90,15 @@ above and completing the kubeadm solution for creating HA clusters.
 - Security: provide a solution that enables the secure copy of cluster certificates between
   control-plane nodes
 
+### Non Goals
+
+- Handle certificate copy in case of External CA.
+  If the user decide to provide kubeadm with an external CA - but without providing the
+  CA keys -, the user will be required to provide signed certificates and all
+  the kubeconfig files (including X509 certificates) not only on the bootstrap master as of today,
+  but also in the joining control-plane nodes.
+  (kubeadm can't generate any certificate without the required CA keys)
+
```

Comment:
A slight change of the wording here in order to make it a bit easier to read:
```
If users decide to use an external CA with kubeadm without providing the CA keys, they will be required to create signed certificates and all the kubeconfig files (including X.509 certificates) for the control plane nodes.
This is needed, because kubeadm cannot generate any certificates without the required CA keys.
```